mne/decoding/tests/test_search_light.py

```diff
@@ -101,10 +101,13 @@ def test_SearchLight():
         assert_array_equal(score_manual, score_sl)
 
     # n_jobs
-    sl = _SearchLight(LogisticRegression(), n_jobs=2)
-    sl.fit(X, y)
+    sl = _SearchLight(LogisticRegression(random_state=0), n_jobs=1,
+                      scoring='roc_auc')
+    score_1job = sl.fit(X, y).score(X, y)
+    sl.n_jobs = -1
```

Comment:
I don't like this -1. Imagine you have a machine with 60 CPUs. Setting it to 2 should be enough for testing.